### PR TITLE
fixing running with CO2Store + water + gas + AquiferCarterTracy

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

File: opm/simulators/aquifers/AquiferCarterTracy.hpp [hunk 1]
```diff
@@ -223,8 +223,17 @@ class AquiferCarterTracy : public AquiferAnalytical<TypeTag>
...
+             Scalar waterViscosity = 0.;
+             if (FluidSystem::phaseIsActive(FluidSystem::oilPhaseIdx)) {
+                 Scalar rs = 0.0; // no dissolved CO2
+                 waterViscosity = FluidSystem::oilPvt().viscosity(pvtRegionIdx(), temp, press, rs);
+             } else if (FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx)) {
+                 Scalar salt = 0.;
+                 Scalar rsw = 0.;
+                 waterViscosity = FluidSystem::waterPvt().viscosity(pvtRegionIdx(), temp, press, rsw, salt);
+             } else {
+                 OPM_THROW(std::runtime_error, "water or oil phase is needed to run CO2Store.");
```
Comment (GitPaean) [inline opm/simulators/aquifers/AquiferCarterTracy.hpp:235]: I tried to do some sanity checking in the constructor functions for AquiferInterface, but somehow `FluidSystem::phaseIsActive()` does not work there. Not sure why.

**Comment by GitPaean** [comment]
It needs some update from the opm-common.

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by totto82** [comment]
Thanks. The code looks reasonable and it fixes the problem.

**Pull request merged**